search-app/src/cmr/search/api/variables.clj

```diff
@@ -56,10 +56,6 @@
   (variable-api-response
    (variable-service/dissociate-variable-to-collections context variable-concept-id body)))
 
-(defn search-for-variables
```

Comment:
I'm guessing this wasn't used anywhere? (I don't see a removal of its reference ...)

Reply:
Yes, this should have been dropped when I moved the search endpoint to routes.clj.